data-processing/common/upload_entities.py

```diff
@@ -1,8 +1,12 @@
+import dataclasses
+import json
 import logging
+import os
 from typing import Any, Dict, List, Optional, Tuple
 
 from peewee import fn
 
+from common.commands.database import OutputDetails
```

Comment:
nit pick: the code base tends to only import from `common` into `common.commands` modules, and not vice versa, to avoid cases of circular dependencies. Could this OutputDetails be moved into the 'upload_entities.py' file?

Reply:
Yes!